statsforecast/mstl.py

```diff
@@ -15,16 +15,20 @@ def mstl(
     x: np.ndarray,  # time series
     period: Union[int, List[int]],  # season length
     blambda: Optional[float] = None,  # box-cox transform
-    iterate: int = 1,  # number of iterations
+    iterate: int = 2,  # number of iterations
     s_window: Optional[np.ndarray] = None,  # seasonal window
-    stl_kwargs: Optional[Dict] = dict(),
+    stl_kwargs: Dict = dict(),
 ):
     if s_window is None:
         s_window = 7 + 4 * np.arange(1, 7)
     origx = x
     n = len(x)
-    msts = [period] if isinstance(period, int) else period
-    iterate = 1
+    if isinstance(period, int):
+        msts = [period]
+    else:
+        msts = sorted(period)
+    if len(msts) == 1:
+        period = 1
```

Comment:
i think this part is unnecessary (we don't use `period` after defining `msts`). non-blocking comment. :)

Reply:
That should be `iterate`. I forgot to export but the check passed, I'll investigate that.